counsel.el

```diff
@@ -4517,6 +4517,82 @@ If there is no such buffer, start a new `shell' with NAME."
                             (reusable-frames . visible)))
     (shell name)))
 
+;;** `counsel-firefox-bookmarks'
+(ivy-set-actions
+ 'counsel-firefox-bookmarks
+ `(("n" ,(lambda (x) (kill-new (second x))) "copy name")
```

Comment:
Please do not use `cl.el` functions here and elsewhere in the PR; see [`(cl) Organization`](https://www.gnu.org/software/emacs/manual/html_node/cl/Organization.html).

I recommend using built-ins such as `car`, `cadr`, `caddr`, etc. or `nth` for clarity. If you think CL functions are clearer than `nth`, then use the ones defined in `cl-lib.el`, e.g. `cl-first`, `cl-second`, etc. as `counsel-org-entity` does.

Reply:
Why add two "copy" actions "n" and "l" when there is already a default "copy" action under "w"?

Reply:
done

Reply:
@tangxinfa I wasn't requesting you remove the "n" and "l" actions, I was just wondering whether we can combine one or the other with the default "w" action, in order to save a letter. I realise now that users may be used to copying the whole line with "w", so maybe we shouldn't override it. Why did you remove the "n" and "l" actions?